easybuild/easyconfigs/f/FSON/FSON-1.0.5-foss-2020b.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = "CMakeMake"
+
+name = 'FSON'
+version = '1.0.5'
+
+homepage = "https://github.com/josephalevin/fson"
+description = "Fortran 95 JSON Parser"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
Thanks for your contribution, @DirectorSloan. I noticed that you used `GCCcore` in https://github.com/easybuilders/easybuild-easyconfigs/pull/13863, but bumped the toolchain to `foss` here. Is there any particular reason for this? If this is not necessary (i.e. if it doesn't need things like OpenBLAS and OpenMPI), it's better to stick to `GCCcore`, as it allows the software to be used by other toolchains (e.g. intel) as well.

Reply:
Thanks for the hint, I'll not aware of that it's needed. I'll try with `GCCCore` again.

Reply:
As of this I will close this PR